README.md

```diff
@@ -1,12 +1,12 @@
 # UDES automated unit test runner
 
-[![Build Status](https://travis-ci.org/unipartdigital/udes-tester.svg?branch=master)](https://travis-ci.org/unipartdigital/udes-tester)
+[![Build Status](https://travis-ci.org/unipartdigital/udes-tester.svg?branch=13.0)](https://travis-ci.org/unipartdigital/udes-tester)
 
 This is an adaptation of
-[`unipartdigital/odoo-tester`](https://github.com/unipartdigital/odoo-tester),
+[`unipartdigital/odoo-tester:13.0`](https://github.com/unipartdigital/odoo-tester),
 modified to use the [UDES fork](https://github.com/unipartdigital/odoo)
```

Comment:
Nit: For UDES fork maybe we should link to `https://github.com/unipartdigital/odoo/tree/udes-13.0`, as default branch is `udes-11.0`.

Reply:
oh, good point. I've updated the link